chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/JTestGroup.java

```diff
@@ -0,0 +1,64 @@
+package com.griddynamics.jagger.user.test.configurations;
+
+import java.util.List;
+
+/**
+ * Group of test which that should be run simultaneously.
+ */
+public class JTestGroup {
+    private String id;
+    private List<JTest> tests;
+
+
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    private JTestGroup(Builder builder) {
+        this.tests = builder.tests;
+        this.id = builder.id;
+    }
+
+
+    public static class Builder {
+        private String id;
+
+        private List<JTest> tests;
+
+        private Builder() {
+        }
+
+        /**
+         * The group of test which should be run simultaneously.
+         *
+         * @param tests a list of test.
+         */
+        public Builder withTests(List<JTest> tests) {
+            this.tests = tests;
+            return this;
+        }
+
+        /**
+         * Sets id or group name.
```

Comment:
"...id or group name" - so, when doe it set id and when a group name? I would expect less ambiguous documentation.